### DISCUSSION TITLE
Fork

### DISCUSSION THREAD

**Discussion by prof-cocoon** [marcel-licence/esp32_basic_synth#97] [Show and tell]
Hi Marcel, I've made some few changes and added some features to your original code. Do you think it's ok to upload them in a github fork?

Happy weekend!
Nicolas.

**Comment by marcel-licence** [comment]
Hi Nicolas,

that sounds nice. Yes sure feel free to create a fork.
Sooner or later I will move the main project to / into https://github.com/marcel-licence/ml_synth_basic_example to have a more platform independent base.

Have a nice weekend too!

Marcel

**Reply by prof-cocoon** [reply]
this is the link to the code
[esp32_prophet](https://github.com/prof-cocoon/esp32_prophet)

I'm still having strong alias noises...it's a topic I don't understand well enough. I'm reading about different methods to deal with it but I don't have the knowledge to merge those codes in your code. I'm more a code tinkerer/scavenger only beggining with dsp.
